### ISSUE TITLE
Dockered client doesn't find hashcat inside container

### ISSUE THREAD

**Issue by unclesp1d3r** [unclesp1d3r/CipherSwarmAgent#32]
The current docker image that we publish does not seem to find the copy of hashcat pre-installed in the container. It doesn't generate an error, but it outputs the following error when it tries to run the initial benchmark:
```
INFO cipherswarm-agent: Performing benchmarks
ERRO cipherswarm-agent: Failed to start benchmark session error="couldn't start hashcat: exec: no command"
```

**Comment by unclesp1d3r** [comment]
This is an issue with the client assuming that the hashcat binary should be called hashcat.bin, but its called hashcat on the image. The work around is to add the environmental variable when running the docker container:

```bash
docker run -e API_URL="http://192.168.4.52" -e API_TOKEN="92ztnN32pKQ2FvNHKnSUTgGj" -e HASHCAT_PATH="/usr/local/bin/hashcat" ghcr.io/unclesp1d3r/cipherswarmagent:pocl
```

**Comment by unclesp1d3r** [comment]
Resolved by commit 118156a
